recipes/aom/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.29.1"
+
+class Aom(ConanFile):
+    name = "aom"
+    description = "AV1 Codec Library"
+    topics = ("conan", "av1", "codec", "video", "encoding", "decoding")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://aomedia.googlesource.com/aom"
+    license = "BSD-2-Clause"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "assembly": [True, False]
+              }
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "assembly": False
+                      }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        # Check compiler version
+        compiler = str(self.settings.compiler)
+        compiler_version = tools.Version(self.settings.compiler.version.value)
+
+        minimal_version = {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "5",
+            "apple-clang": "6"
+        }
+        if compiler not in minimal_version:
+            self.output.warn(
+                "%s recipe lacks information about the %s compiler standard version support" % (self.name, compiler))
+        elif compiler_version < minimal_version[compiler]:
+            raise ConanInvalidConfiguration("{} requires a {} version >= {}".format(self.name, compiler, compiler_version))
+
+
+    def build_requirements(self):
+        if self.options.assembly:
+            self.build_requires("nasm/2.15.05")
+        if tools.os_info.is_windows and not tools.which("perl"):
+            self.build_requires("strawberryperl/5.30.0.1")
```

Comment:
```suggestion
        if tools.os_info.is_windows:
            self.build_requires("strawberryperl/5.30.0.1")
```

Prefer using the build_requirement for Perl, otherwise, any build problem will be hard to reproduce if someone open an issue. With a build requirement, we can use exactly same perl version.